lineman/app/components/thread_page/activity_card/new_comment.haml

```diff
@@ -1,4 +1,4 @@
-%article.thread-item.thread-item--comment{ng_controller: 'NewCommentItemController', aria-label: "{{ 'activity_card.new_comment.aria_label' | translate }} {{ ::comment.authorName() }}"}
```

Comment:
this is probably not good enough namespacing.. i think you'll need type-id rather than just id. I'll show you what i mean shortly